x-pack/plugins/ml/public/application/settings/filter_lists/list/table.js

```diff
@@ -213,7 +214,7 @@ export function FilterListsTable({
             isSelectable={true}
             data-test-subj="mlFilterListsTable"
             rowProps={(item) => ({
-              'data-test-subj': `mlCalendarListRow row-${item.filter_id}`,
+              'data-test-subj': `mlFilterListsTable row-${item.filter_id}`,
```

Comment:
```suggestion
              'data-test-subj': `mlFilterListsRow row-${item.filter_id}`,
```
We already use `mlFilterListsTable` for the whole table. This test subject is for the row only.

Reply:
Updated here https://github.com/elastic/kibana/pull/76735/commits/dc3ba76792563da7c40db99ac11c6d69100f3bd7